knimin/lib/parse.py

```diff
@@ -0,0 +1,32 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The LabAdmin Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+
+
+def parse_plate_reader_output(contents):
```

Comment:
I've described the format. I've been talking with the Rodolfo and Greg and they will be generating the simplest file possible.